templates/kubernetes/terraform/modules/kubernetes/files/kratos-values.yml

```diff
@@ -0,0 +1,160 @@
+image:
+  repository:  oryd/kratos
+  # tag: v0.4.6-alpha.1-sqlite
+  pullPolicy: IfNotPresent
+
+ingress:
+  public:
+    enabled: true
+    annotations:
+        kubernetes.io/ingress.class: nginx
+        # nginx.ingress.kubernetes.io/rewrite-target: /$1
+        cert-manager.io/cluster-issuer: clusterissuer-letsencrypt-production
+    hosts: # Filled by user_auth.tf
+      - paths: ["/"]
+    tls:
+      - secretName: kratos-public-tls-secret
+
+  admin:
+    enabled: true
+    annotations:
+        kubernetes.io/ingress.class: nginx
+        # nginx.ingress.kubernetes.io/rewrite-target: /$1
+        cert-manager.io/cluster-issuer: clusterissuer-letsencrypt-production
+    hosts: # Filled by user_auth.tf
+      - paths: ["/"]
+    tls:
+      - secretName: kratos-admin-tls-secret
+
+kratos:
+  development: false
+  autoMigrate: true
+
+#  You can add multiple identity schemas here
+  identitySchemas:
+    "identity.default.schema.json": |
+      {
+        "$id": "default",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "title": "Person",
+        "type": "object",
+        "properties": {
+          "traits": {
+            "type": "object",
+            "properties": {
+              "email": {
+                "type": "string",
+                "format": "email",
+                "ory.sh/kratos": {
+                  "credentials": {
+                    "password": {
+                      "identifier": true
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+  config:
+    secrets:
+      default: ["test-insecure-session-id-here"]
+    courier:
+      smtp:
+        connection_uri: "http://mail-server" # TODO: Mail support
+
+
+    hashers:
+      argon2:
+        parallelism: 1
+        memory: 131072
+        iterations: 2
+        salt_length: 16
+        key_length: 16
+
+    identity:
+      default_schema_url: file:///etc/config/identity.default.schema.json
+
+    selfservice:
+      methods:
+        password:
+          enabled: true
+        oidc: # TODO: allow methods to be configured with parameters
+          enabled: true
+          # config:
+          # TODO: use SELFSERVICE_METHODS_OIDC_CONFIG_PROVIDERS to set via secrets
+          # [{"id":"github","provider":"github","client_id":"id","client_secret":"secret","mapper_url":"http://zombism.ca/github.data-mapper.jsonnet","scope":["user:email"]}]
+            # providers:
+            #   - id: github
+            #     provider: github
+            #     client_id: # TODO: load secret
+            #     client_secret: # TODO : load secret
+            #     # TODO: Change to static or deploy somewhere within infra
+            #     mapper_url: http://zombism.ca/github.data-mapper.jsonnet
+            #     scope:
+            #       - user:email
+
+      default_browser_return_url: /dashboard
```

Comment:
we may have have to have the entire frontend domain in all of these paths or else it redirects to the backend instead of back to the UI

Reply:
Yeah, need to tweak those for sure.